package.json

```diff
@@ -14,7 +14,7 @@
     "build-integration-tests": "cd ./integration-tests && npm i",
     "check-spelling": "npx cspell **/*.*",
     "clean-build": "lerna run clean-build",
```

Comment:
But, if clean is pulled to the top, then `clean-build` would become: `npm run clean && npm run build`.